Task 7/routes/mongo/cityRouter.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const City = require('../../mongoose/models/city.js');
```

Comment:
Consider creating an 'index.js' file in the models folder, importing all models there - and immediately exporting then again.
That makes it possible to import like so: "const { City } = require('../../mongoose/models');"